docusaurus.config.js

```diff
@@ -76,11 +76,14 @@ const config = {
         theme: {
           customCss: require.resolve('./src/css/custom.css'),
         },
+        gtag: {
+          trackingID: 'G-G856J3S8ZG',
+          anonymizeIP: true,
+        }
```

Comment:
Make sure you have the trailing comma as is the rest of the style for the site. It helps with maintainability.